### PR TITLE
android pvc check

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/AGDIGGER-133

**Changes**
- Add Nagios Check on pvc
- Description Improvements

### CODE REVIEW

File: fhservices.cfg.j2 [hunk 1]
```diff
@@ -226,9 +230,17 @@ define service {
...
-      service_description android_sdk:ping
+      service_description android_sdk::ping
       check_command android_sdk_health
       use generic-service
+      hostgroup_name none
+      contact_groups rhmapadmins
+}
+
+define service {
+      service_description android_pvc:check
```
Comment (wei-lee) [inline fhservices.cfg.j2]: double `:`?

Comment (laurafitzgerald) [inline fhservices.cfg.j2]: yep :smile:

**Comment by laurafitzgerald** [comment]
retest this please

**Comment by laurafitzgerald** [comment]
restest this please

**Comment by laurafitzgerald** [comment]
retest this please

**Comment by laurafitzgerald** [comment]
**Verification**
- Changes are deployed to OSM4 in project [digger](https://osm4.skunkhenry.com:8443/console/project/digger/overview)
- Check that the status of the android-sdk pvc check is OK

- Deploy image `docker.io/rhmap/nagios4:4.1.0-218` to local digger deployment
- oc edit pvc android-sdk - set spec.status to Failed
- check that the status of the android-sdk pvc check is CRIT

**Comment by odra** [comment]
👀

**Comment by odra** [comment]
👍 Verified it locally

**Comment by laurafitzgerald** [comment]
As discussed in standup I am merging the changes as they are reviewed and verified. Will add the tests later.

**Pull request merged**